### PR TITLE
MP CORS doc update

### LINKED ISSUE #4201: [Doc:] Helidon MP CORS
Helidon MP CORS documentation (according to the new documentation structure).

Path: `/HelidonMP/Security`

Sub-chapters should be united into one.

- [x] Overview (may be reused from current doc)
- [x] Maven Dependency (may be reused from current doc)
- [x] Usage (may be reused from current doc)
- [x] API (may be reused from current doc)
- [x] Configuration (may be reused from current doc)
- [x] Reference (generated)
- [x] Examples (may be reused from Helidon repo or current doc)
- [x] Additional Information (if applicable)

### PR DESCRIPTION
**Pull request by tjquinno**
Resolves #4201 

Mostly focused on MP CORS. This includes a few minor changes to the common CORS file and the SE CORS file.

### CODE REVIEW

File: docs/mp/cors/cors.adoc [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+The index `_i_` is an integer (0, 1, 2, etc.).
+<3> Specify the settings as needed to define the CORS behavior you want for that resource.
+<4> The `enabled` setting lets you control whether the system uses that set of CORS configuration.
+
+The system uses the index `_i_`, not the position in the config file, to identify the settings for a particular resource.
+
+Path patterns can be any expression accepted by the link:{webserver-javadoc-base-url}/io/helidon/webserver/PathMatcher.html[`PathMatcher`] class.
+
+NOTE: Be sure to assign index values to the entries in the order that you want Helidon to check them.
+Helidon CORS support searches the cross-origin entries in index order them until it finds an entry that
```
Comment (ljamen) [inline docs/mp/cors/cors.adoc:128]: Not sure what this means? 
Helidon CORS support searches the cross-origin entries in index order them until ...

**Review by ljamen — COMMENTED** [review]
Changes look good to me. One typo (?) maybe.

File: docs/mp/cors/cors.adoc [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+cors.paths._i_.allow-origins=
+cors.paths._i_.expose-headers=
+cors.paths._i_.allow-methods=
+cors.paths._i_.enabled= # <4>
+----
+<1> You can disable CORS processing for all resources by setting `cors.enabled` to `false`. Defaults to `true`.
+<2> Add a block for each resource you want to configure.
+The index `_i_` is an integer (0, 1, 2, etc.).
+<3> Specify the settings as needed to define the CORS behavior you want for that resource.
+<4> The `enabled` setting lets you control whether the system uses that set of CORS configuration.
```
Comment (spericas) [inline docs/mp/cors/cors.adoc:121]: Should we say default is `true` here too?

**Review by spericas — DISMISSED** [review]

File: docs/mp/cors/cors.adoc (hunk 2 above)
Comment (tjquinno) [reply docs/mp/cors/cors.adoc]: Done.

**Review by spericas — APPROVED** [review]

File: docs/mp/cors/cors.adoc (hunk 2 above)
Comment (tjquinno) [reply docs/mp/cors/cors.adoc]: Done

**Pull request merged**